jekyll/_cci2/language-ruby.md

```diff
@@ -38,8 +38,159 @@ Database images for use as a secondary 'service' container are also available on
 
 ## Sample Configuration
 
+{:.tab.rubyconf.Config_2__1}
 {% raw %}
 
+```yaml
+version: 2.1 # Use CircleCI 2.1 
+orbs: # use orbs to help shorten and reduce repetition in our config.
+  ruby: circleci/ruby@0.1.2 
```

Comment:
We can use the latest version of this orb!

```suggestion
  ruby: circleci/ruby@0.2.1
```

Reply:
The repository is still using the `0.1.2` version, so this is not a necessary change.